src/survey/constants.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export const BannerActions = {
+  bannerLabelYes: 'Take Survey',
+  bannerLabelNo: 'Don\'t show again',
+  remindLater: 'Remind Me later'
+}
+
+export const SurveyBanner = {
+  message: 'Do you mind taking a quick feedback survey about the Java extension pack?',
+  action: [
+    BannerActions.bannerLabelYes,
+    BannerActions.bannerLabelNo,
+    BannerActions.remindLater
+  ],
+  FIRST_STAGE_URL: 'https://www.microsoft.com',
```

Comment:
@akaroml @yaojinyang Do we have the survey links for the three different stages?